how-to/author-guide.md

```diff
@@ -268,23 +271,34 @@ are usually not accepted by JOSS. Be sure to review JOSS's
+
+### Ongoing support from pyOpenSci
+
+We're committed to supporting you throughout your package's maintenance journey:
 
-Once you have been accepted into the pyOpenSci ecosystem, you will
-be invited to join our Slack channel.
+**Annual check-ins**: We'll reach out each year to see how your package is doing and learn about any updates we can help highlight through our blog, social media, or newsletter.
 
-We also will keep in touch with you, periodically checking in to ensure that package maintenance is going well and to better understand ways in which pyOpenSci can support you.
+**Community resources**: Take advantage of our Slack community to connect with other maintainers, share experiences, and get advice on common maintenance challenges. We're also building additional resources and tools to help package maintainers succeed.
 
-If at any time, you need to step down from maintaining your package,
-or you need help with maintenance, please let us know - preferably
-in advance. We will try to help you by either:
+**Maintenance assistance**: If you have specific maintenance challenges, our Slack community is a great place to ask questions. There is a lot of packaging and community expertise in our vibrant community!
 
-- Finding a new maintainer to take over your project (or additional maintainers to support maintenance) or
-- Sunsetting your package.
+### When you need help or want to step down
 
-If the package is sunsetted, we will remove it from our curated list
+Life changes and priorities shift. If maintaining your package becomes challenging, please reach out to us. We're here to help, and we have several options:
+
+- **Finding co-maintainers or new maintainers**: If you are interested, we can try to connect you with community members interested in contributing to or taking over maintenance of your package.
+- **Package archival**: If maintenance isn't sustainable, we can work together [to archive your package](https://www.pyopensci.org/python-packages.html#archived-packages).
+
+If the package is archived, we will remove it from our curated list
 of vetted tools.
 
-### Communication with pyOpenSci and removing tools from our vetted tool list
+### Maintenance policies
 
-To ensure packages that we support and advocate are maintained, if your package is accepted and we are not able to get in touch with you through normal communication channels (GitHub, email) after reaching out for at least 1-2 months, we will remove your package from our list of vetted tools. We will also remove any blogs written that highlight your tool.
+For detailed information about our maintenance expectations, annual check-in process, and archival procedures, please take a look at our [post-review process documentation](post-review-process). Remember, you maintain full ownership of your package. We're here to support you, not interfere with your development process.
```

Comment:
This also addresses some of the concerns around pyOpenSci "owning" a package. We never intended that to be implied in our documentation. So I just softened the language here!